candid/evm_rpc.did

```diff
@@ -226,14 +227,14 @@ type ValidationError = variant {
 service : (InitArgs) -> {
   authorize : (principal, Auth) -> ();
   deauthorize : (principal, Auth) -> ();
-  eth_feeHistory : (RpcSource, FeeHistoryArgs) -> (MultiFeeHistoryResult);
-  eth_getBlockByNumber : (RpcSource, BlockTag) -> (MultiGetBlockByNumberResult);
-  eth_getLogs : (RpcSource, GetLogsArgs) -> (MultiGetLogsResult);
-  eth_getTransactionCount : (RpcSource, GetTransactionCountArgs) -> (
+  eth_feeHistory : (RpcSource, opt RpcConfig, FeeHistoryArgs) -> (MultiFeeHistoryResult);
```

Comment:
Just wondering... why don't we follow the pattern of a single argument for these functions?
For example, why not have `RpcSource` and `RpcConfig` inside `FeeHistoryArgs`?

Reply:
`RpcSource` and `RpcConfig` are expected for all Candid-RPC methods, whereas everything in `FeeHistoryArgs` is specific to the `eth_feeHistory` JSON-RPC method. I suppose we could include `source` and the fields in `config` in a record arg, but this would lead to an awkward situation if the RPC method also includes something named `source` or `config`. 

I might give this a try anyway in case there ends up being a relatively nice way to do this.